release-notes.txt

```diff
@@ -1,3 +1,30 @@
+OpenMDAO Version 1.7.0 Alpha Release Notes
+June 30, 2016
+
+Backwards-Incompatible Change:
+* Change fd_options to be more self consistent, changed name to deriv_options.
+
+Features:
+* NAS access component
+* Derivatives output adds an asterisk to the 'Absolute Error' and 'Relative Error'
```

Comment:
The output of `check_partial_derivatives` has been modified so that absolute or relative errors that exceed a tolerance are flagged with a '*'. The tolerances can be specified via call arguments.